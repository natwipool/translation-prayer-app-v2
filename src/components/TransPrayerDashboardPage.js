import React from 'react';
import TransPrayerList from './TransPrayerList';

const TransPrayerDashboardPage = () => (
  <div>
    <div className="page-header">
      <div className="content-container-body">
        <h2 className="page-header__title">สารบัญ</h2>
      </div>
    </div>
    <TransPrayerList />
  </div>
);

export default TransPrayerDashboardPage;
