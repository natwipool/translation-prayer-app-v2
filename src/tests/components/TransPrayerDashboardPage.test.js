import React from 'react';
import { shallow } from 'enzyme';
import TransPrayerDashboardPage from '../../components/TransPrayerDashboardPage';

test('should render TransPrayerDashboardPage correctly', () => {
  const wrapper = shallow(<TransPrayerDashboardPage />)
  expect(wrapper).toMatchSnapshot();
});