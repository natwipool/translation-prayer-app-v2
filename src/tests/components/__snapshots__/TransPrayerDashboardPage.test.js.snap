// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TransPrayerDashboardPage correctly 1`] = `
<div>
  <div
    className="page-header"
  >
    <div
      className="content-container-body"
    >
      <h2
        className="page-header__title"
      >
        สารบัญ
      </h2>
    </div>
  </div>
  <Connect(TransPrayerList) />
</div>
`;
